### DISCUSSION TITLE
"Add Nilchain To Keplr Wallet" component - Nillion Bounty Builder - Open Source

### DISCUSSION THREAD

**Discussion by tonytrandevv** [NillionNetwork/discussions#109] [Show and tell]
### Show and tell project type

Builder Bounty Submission

### Github Repo Link

https://github.com/tonytrandevv/add-Nilchain-to-Keplr

### Project Description

**A React component allows user to add the Nilchain to Keplr Wallet**

**Overview**
In this project, a react component is provided to intergrate with a npm package that allows developers are easily to customize any blockchain within Cosmos ecosystem and add to Keplr Wallet.

**Live Project**
- Website: https://nillion-add-nilchain-to-keplr.vercel.app/
- Npm package: https://www.npmjs.com/package/add-keplr-button

**Features**
- Adding Nillion chain to Keplr Wallet.
- Allowing developers to customize chain information easily.
- A friendly user interface with a configurable button to connect with Keplr Wallet.
- Handling the chain addition process with errors and feedbacks.
- Compatible with the most popular browsers (Edge, Chrome, Firefox).

### Optional - Link your project and team members' social handles

https://x.com/validachain

### Optional - Team ETH Address(es)

0x10b1BAFc42c90eF5278AAA87BEfcEC68F3E049Cf

**Comment by crypblizz8** [comment]
Hi there, is the Vercel link public?

**Reply by tonytrandevv** [reply]
Here it is : https://nillion-add-nilchain-to-keplr.vercel.app/
